### DISCUSSION TITLE
add user id automatically to a field in the table that has relation with table user belongs to

### DISCUSSION THREAD

**Discussion by stjrna** [thedevdojo/voyager#5475] [Q&A]
![voyager question](https://user-images.githubusercontent.com/63811413/138288913-f07a7b9f-4535-4a97-9efe-5aa3526c8104.png)
Hello All

I tried to google it but found no answer

I need to add user id automatically to a field in the table that has relation with table user belongs to, (so I need to know what user is responsible on the field) 
now what happen is when I do relationship belong to , I can choose from drop down 

(I am not sure if I have to customize a code to do so, and if it is the case where is the code path)
Best regards
Omer

**Comment by stjrna** [comment]
I think I find answer here 👍 I am new with both laravel and voyager but I will try to figure how to apply that 🥇 

[https://voyager-docs.devdojo.com/customization/overriding-files](url)

**Comment by jeancarloscharao** [comment]
Boa tarde, você pode usar Mutator e colocar no model Product   

    public function setUserIdAttribute()
    {
        $this->attributes['user_id']  = Auth::user()->id;
    }

https://laravel.com/docs/8.x/eloquent-mutators

**Reply by stjrna** [reply]
Bon tarde 
mucha gracias
